docs/sources/panels-visualizations/query-transform-data/_index.md

```diff
@@ -19,7 +19,7 @@ labels:
     - enterprise
     - oss
 title: Query and transform data
-weight: 200
+weight: 40
```

Comment:
2 questions, mostly out of curiosity:

- Most updated weights are vastly different, how did you come up with the new values? Also, are these weights for the fonts, I assume?
- Why are some `index.md` files prepended with an underscore, and some are not?

Reply:
Weights control order in the table of contents: https://grafana.com/docs/writers-toolkit/write/front-matter/#weight.

The different index files are Hugo page bundles and are documented in https://gohugo.io/content-management/page-bundles/.